src/attributes.js

```diff
@@ -12,7 +12,10 @@ export class AttributesCustomAttribute {
   }
 
   valueChanged() {
-    $(this.element).attr(this.value || {});
+    Object.keys(this.value).forEach(attr => {
+      let value = this.value[attr];
+      this.element.setAttribute(attr, value);
```

Comment:
Rename attr, and inline the above declared variable. (`this.element.setAttribute(attribute, this.value[attribute])`)